### PR TITLE
fixed the error message for a too short record-period when online lea…

### PR DESCRIPTION
**Pull request by jbaron**
The error message had two small mistakes, the F-String was missing and the message itself was wrong.

### CODE REVIEW

File: pearl/utils/functional_utils/train_and_eval/online_learning.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def online_learning(
         if number_of_steps is not None and episode_total_steps > record_period:
             print(
                 f"An episode is longer than the report_period: episode length {episode_total_steps}"
```
Comment (antoine-galataud) [inline pearl/utils/functional_utils/train_and_eval/online_learning.py:118]: `report_period` -> `record_period`?

Comment (jbaron) [reply pearl/utils/functional_utils/train_and_eval/online_learning.py:118]: Good catch

Comment (jbaron) [reply pearl/utils/functional_utils/train_and_eval/online_learning.py:118]: You prefer a new PR?

Comment (jbaron) [reply pearl/utils/functional_utils/train_and_eval/online_learning.py]: Updated main branch with fix for report_period -> record_period

**Comment by rodrigodesalvobraz** [comment]
Thanks, good catch!

**Pull request closed without merging**